content/blog/posts/for-the-love-of-the-game.md

```diff
@@ -0,0 +1,109 @@
+
+---
+title: "For the Love of the Game: In Search of Best-of-Breed 3D Creation Tools"
+date: 2022-09-16
+slug: for-the-love-of-the-game
+author: Nicole Huesman
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+blog_img: "/images/blog/announcement_thumbnail.jpg"
+
+---
+## For the Love of the Game: In Search of Best-of-Breed 3D Creation Tools
+
+Born out of a desire to make their own games, indie game developers Jacob Dodenhoff and Phoenix Cook forged [Chaotic Creations Interactive](https://www.chaoticcreationsinteractive.com/) (CCI), a **small game development studio** in the mid-west of the United States. Their studio reflects who they are and what they value: the players, and the commitments they make to them. Since the birth of CCI, they’ve been exploring game engines and they discovered the [Open 3D Engine](https://www.o3de.org/) in July 2021 when it was announced. In this blog, they share their experiences and offer insight to newcomers who may be interested in taking the Open 3D Engine for a spin.
+
+![CCI-indie-developers](/static/images/blog/CCI/CCI-blog-image.png)
```

Comment:
Images do not require the `/static` path in the link, and won't render if it's included.

```suggestion
![CCI-indie-developers](/images/blog/CCI/CCI-blog-image.png)
```